crates/ruff_linter/src/rules/flake8_datetimez/rules/call_datetime_fromtimestamp.rs

```diff
@@ -49,14 +48,20 @@ use super::helpers;
 /// ## References
 /// - [Python documentation: Aware and Naive Objects](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)
 #[violation]
-pub struct CallDatetimeFromtimestamp;
+pub struct CallDatetimeFromtimestamp(DatetimeModuleAntipattern);
 
 impl Violation for CallDatetimeFromtimestamp {
     #[derive_message_formats]
     fn message(&self) -> String {
-        format!(
-            "The use of `datetime.datetime.fromtimestamp()` without `tz` argument is not allowed"
-        )
+        let CallDatetimeFromtimestamp(antipattern) = self;
+        match antipattern {
+            DatetimeModuleAntipattern::NoTzArgumentPassed => format!(
+                "The use of `datetime.datetime.fromtimestamp()` without `tz` argument is not allowed"
```

Comment:
is it not allowed (raise an error) or it is just a bad practice?

Reply:
Great question. The error messages we were giving were much more verbose than is normal for our messages, and were still pretty unclear. I tidied them up in https://github.com/astral-sh/ruff/pull/10621/commits/7beb878e274ee091ed2bd7c138906565253c7951.